querier.go

```diff
@@ -328,7 +383,7 @@ func postingsForMatcher(ix IndexReader, m labels.Matcher) (index.Postings, error
 	return index.Merge(rit...), nil
 }
 
-func postingsForUnsetLabelMatcher(ix IndexReader, m labels.Matcher) (index.Postings, error) {
+func inversePostingsForMatcher(ix IndexReader, m labels.Matcher) (index.Postings, error) {
```

Comment:
Can we have some comments here describing it?

> Returns the postings for the series with the label name set but not matching the matcher.

Reply:
Done